Duo/duo/connector.py

```diff
@@ -0,0 +1,69 @@
+import signal
+import time
+from threading import Lock
+
+from sekoia_automation.connector import Connector, DefaultConnectorConfiguration
+from sekoia_automation.storage import PersistentJSON
+
+from . import DuoModule, LogType
+from .consumer import DuoLogsConsumer
+
+
+class AdminLogsConnectorConfiguration(DefaultConnectorConfiguration):
+    frequency: int = 60
+    chunk_size: int = 1000
+
+
+class DuoAdminLogsConnector(Connector):
+    module: DuoModule
+    configuration: AdminLogsConnectorConfiguration
+
+    LOGS_TO_FETCH = (LogType.AUTHENTICATION, LogType.ADMINISTRATION, LogType.TELEPHONY, LogType.OFFLINE)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.context = PersistentJSON("context.json", self._data_path)
+        self.context_lock = Lock()
+        self.consumers = {}
+
+        # Register signal to terminate thread
+        signal.signal(signal.SIGINT, self.exit)
+        signal.signal(signal.SIGTERM, self.exit)
+
+    def exit(self, _, __):
+        self.log(message="Stopping Duo Admin Logs connector", level="info")
+        # Exit signal received, asking the processor to stop
+        self._stop_event.set()
```

Comment:
A parent class of Connector already manage SIGINT and SIGTERM.
```suggestion

    def stop(self, *args, *kwargs):
        self.log(message="Stopping Duo Admin Logs connector", level="info")
        super().stop(*args, **kwargs)
```